src/OVAL/oval_variable.c

```diff
@@ -419,6 +523,39 @@ struct oval_variable *oval_variable_clone(struct oval_definition_model *new_mode
 
 			evar = (oval_variable_EXTERNAL_t *) new_variable;
 			old_evar = (oval_variable_EXTERNAL_t *) old_variable;
+
+			struct oval_iterator *old_pv_itr;
+			old_pv_itr = oval_variable_get_possible_values(old_variable);
+			while (oval_collection_iterator_has_more(old_pv_itr)) {
+				struct oval_variable_possible_value *old_pv, *new_pv;
+				old_pv = oval_collection_iterator_next(old_pv_itr);
+				new_pv = oval_variable_possible_value_new(old_pv->hint, old_pv->value);
+				oval_collection_add(evar->possible_values, new_pv);
```

Comment:
Would be nice if you provided an API to add possible value and then used it here.